app/javascript/Application/util/ApiErrorHandler.js

```
@@ -0,0 +1,26 @@
+import { globalAlertService } from './GlobalAlertService';
+
+const ApiErrorHandler = {
+  handleError,
+};
+
+function postErrors(issueDetails) {
+  issueDetails.forEach(detail => {
+    let message = detail.user_message;
+    message += detail.incident_id ? ' [Incident id:' + detail.incident_id + ']' : '';
+    globalAlertService.postError({ message: message });
+  });
+}
+
+export function handleError(error) {
+  if (error.response && error.response.data) {
+    let data = error.response.data;
+    postErrors(data.issue_details || []);
```

What if the is no `data.issue_details`? Will the error be lost?

---

it will pass error.message